BALSAMIC/commands/config/case.py

```diff
@@ -117,6 +117,12 @@
 @click.option(
     "--normal-sample-name", type=str, required=False, help="Normal sample name"
 )
+@click.option(
+    "--cadd-annotations",
```

Comment:
As this cadd annotation file is only used to annotate InDels, I think that could be made clearer, either in the help text or in the name itself. To something like this: 
```suggestion
    "--indel-cadd-annotations",
```